app/middlewares/errors.js

```diff
@@ -6,7 +6,9 @@ const DEFAULT_STATUS_CODE = 500;
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
   [errors.DEFAULT_ERROR]: 500,
-  [errors.BAD_REQUEST_ERROR]: 400
+  [errors.BAD_REQUEST_ERROR]: 400,
+  [errors.UNAUTHORIZED_ERROR]: 401,
+  [errors.AUTHENTICATION_ERROR]: 403
```

Comment:
AUTHENTICATION_ERROR should be 401 and UNAUTHORIZED_ERROR (forbidden is better) 403 😛

Reply:
Sorry. I understand now, I have replaced it.😅😅😅😅